.github/workflows/manual-test.yaml

```
@@ -0,0 +1,76 @@
+name: Manual Test
+
+on:
+  workflow_dispatch:
+    inputs:
+      conda-version:
+        description: 'Conda Version'
+        required: true
+        default: '23.1.0'
+        type: choice
+        options:
+        - 'main'
+        - '23.1.0'
+      conda-libmamba-version:
+        description: 'conda-libmamba-solver version/branch'
+        required: true
+        default: '23.1.0'
+      channel:
+        description: 'Conda Channel'
+        required: true
+        default: 'conda-forge'
+      os:
+        description: 'Operating System'
+        required: true
+        default: 'ubuntu-latest'
+        type: choice
+        options:
+          - macos-latest
+          - ubuntu-latest
+          - windows-latest
+
+jobs:
+  conda-libmamba-solver:
+    name: conda-libmamba-solver=${{ inputs.conda-libmamba-version }} os=${{ inputs.os }}
+    runs-on: ${{ inputs.os }}
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@master
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-activate-base: true
+          activate-environment: ""
+          conda-version: ${{ inputs.conda-version }}
+
+      - name: conda-libmamba-solver from Conda
+        shell: bash -el {0}
+        run: conda install -c conda-forge conda-libmamba-solver==${{ inputs.conda-libmamba-version }}
+        if: ${{ startsWith(inputs.conda-libmamba-version, '2') }}
```

```suggestion
        if: ${{ startsWith(inputs.conda-libmamba-version, '2') }}
```

This will fail in 2030! jk jk